frontend/src/components/AboutUs.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+    <div class="cfp-background">
+        <v-container fluid text-xs-center>
+            <v-layout row class="cfp-deco-title cfp-font-title">
+                <v-flex> 
+                    <v-card flat dark tile color="transparent">
+                        <v-card-text>
```

Comment:
Just curious - why didn't you use v-card-title for the title? :D

Reply:
Mainly because I wasn't aware of it's existence :) Isn't that just a matter of approach?